lua/telescope/utils.lua

```diff
@@ -251,6 +251,10 @@ utils.diagnostics_to_tbl = function(opts)
   return items
 end
 
+utils.path_shorten = function(file)
```

Comment:
We should have kept this as shorthand, but make it work for the new interface

Reply:
That's probably the biggest one, yes.